### DISCUSSION TITLE
PS1 Internal Game Save

### DISCUSSION THREAD

**Discussion by dezquerro** [OnionUI/Onion#637] [Q&A]
Hi all,

I am having some issues with the internal save features on PS1 (that I am aware of).
I am playing Castlevania SotN and, when I save in-game (in the regular saving spots) and I die, I am not able to go back to the saving spot. It is as if the game didn't actually save.
I have to exclusively use save states with PS1 because the in-game save feature is not working properly.

Thanks in advance for your support.
Best,

Diego

**Comment by Aemiii91** [comment]
This might be fixed by disabling SMC checks.

Go to RetroArch Quick Menu (MENU+SELECT while in-game), select Options and toggle on `(Speed Hack) Disable SMC Checks`.
